2-page-structure/examples.js

```diff
@@ -1,10 +1,15 @@
-// example of a React fragment (shorthand) without any props or keys on the Fragment tag
-
-function HeadingItems({ item }) {
-  return (
+// Good - correct structure
+const HeadingItemsGood = ({ item }) => (
     <>
       <h1>{item.term}</h1>
       <h2>{item.description}</h2>
     </>
-  );
-}
+);
+
+// Bad - not semantic, can only use one <h1> per page
```

Comment:
Could add here to explain that it is not semantic because the description would not belong in an h1 tag, as well as only using 1 h1 tag